db/migrate/20150623021304_add_null_false_for_formatted_address_on_addresses_table.rb

```diff
@@ -0,0 +1,8 @@
+class AddNullFalseForFormattedAddressOnAddressesTable < ActiveRecord::Migration
+  def change
+    change_table :addresses do |t|
```

Comment:
Unused block argument - `t`. You can omit the argument if you don't care about it.